README.md

```diff
@@ -1,19 +1,76 @@
 # Kashmir [![NPM version][npm-image]][npm-url]
-> Generator for the Kashmir Drupal theme
+> A Drupal theme generator
 
 ## Installation
 
-First, install [Yeoman](http://yeoman.io) and generator-kashmir using [npm](https://www.npmjs.com/) (we assume you have pre-installed [node.js](https://nodejs.org/)).
+Install [Yeoman](http://yeoman.io) and generator-kashmir using [npm](https://www.npmjs.com/) (we assume you have pre-installed [node.js](https://nodejs.org/)).
+
 
 ```bash
 npm install -g yo generator-kashmir
```

Comment:
This would now be `npm install -g generator-kashmir`
*generator-kashmir* would be replaced with the new package name.